osf/models/node.py

```diff
@@ -459,12 +464,23 @@ def is_preprint_orphan(self):
     def has_published_preprint(self):
         return self.preprints.filter(is_published=True).exists()
 
+    @property
+    def linked_preprint(self):
+        return self.preprints.get_queryset()[0]
+
+    @property
+    def has_moderated_preprint(self):
+        return self.preprints.filter(provider__reviews_workflow__isnull=False).exists()
```

Comment:
Instead of `has_moderated_preprint`, couldn't the view just check `node.linked_preprint.provider.reviews_workflow`? This seems like an unnecessary query.